src/System.Security.Cryptography.X509Certificates/tests/X509StoreTests.cs

```diff
@@ -496,5 +499,48 @@ public static void UnixCannotModifyDisallowedStore(bool useEnum, OpenFlags openF
                 Assert.Equal(0, store.Certificates.Count);
             }
         }
+
+        [Fact]
+        [PlatformSpecific(TestPlatforms.AnyUnix)]
```

Comment:
Not sure what this will do on OSX, but giving it a chance in CI.

Reply:
Can you also make `[OuterLoop]` ? Either after getting CI results or you will have to explicitly request CI do an outerloop run.

Reply:
If the reason for outerloop is for cleanup, the test cleans up nicely. Or is it for other reasons?

Reply:
If it cleans up - and sets no global state - probably not. @stephentoub you mentoined outerloop.

Reply:
how does the file you did chmod to get cleaned up?

Reply:
All files created are in `sslCertDir` which gets cleaned-up by the `FileCleanupTestBase` base class.